pkgs/tools/misc/woof/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchurl, python }:
+
+stdenv.mkDerivation rec {
+  version = "2012-05-31";
+  name = "woof-${version}";
+  
+  src = fetchurl {
+    url = "http://www.home.unix-ag.org/simon/woof-${version}.py";
+    sha256 = "d84353d07f768321a1921a67193510bf292cf0213295e8c7689176f32e945572";
+  };
+
+  buildInputs = [ python ];
+  runtimeDeps = [ python ];
+
+  unpackPhase = "true";
+
+  installPhase =
+    ''
+      mkdir -p $out/bin
+      cp $src $out/bin/woof
+      chmod +x $out/bin/woof
+    '';
+
+  meta = {
+    homepage = http://www.home.unix-ag.org/simon/woof.html;
+    description = "Web Offer One File - Command-line utility to easily exchange files over a local network";
+    license = stdenv.lib.licenses.gpl2Plus;
+    platforms = stdenv.lib.platforms.unix;
+  };
```

Comment:
To add yourself as a maintainer, use
```
maintainers = with stdenv.lib.maintainers; [ lschuermann ];
```
And don't forget to add yourself to https://github.com/NixOS/nixpkgs/blob/master/lib/maintainers.nix

Reply:
If you want to leave out all that `stdenv.lib`, you can do it like that:
```
meta = with stdenv.lib; {
  homepage = http://www.home.unix-ag.org/simon/woof.html;
  description = "Web Offer One File - Command-line utility to easily exchange files over a local network";
  license = licenses.gpl2Plus;
  platforms = platforms.unix;
  maintainers = with maintainers; [ lschuermann ];
};